dataactcore/models/errorInterface.py

```diff
@@ -82,6 +82,9 @@ def checkNumberOfErrorsByJobId(self, jobId):
         queryResult = self.session.query(ErrorMetadata).filter(ErrorMetadata.job_id == jobId).all()
         numErrors = 0
         for result in queryResult:
+            if result.severity_id != valDb.getRuleSeverityId(errorType):
+                # Don't count non-fatal
```

Comment:
Yep, the comment was from the first version of the function before it expanded to handle warnings as well, rephrased to be more general.